akka-distributed-data/src/multi-jvm/scala/akka/cluster/ddata/DurableDataSpec.scala

```diff
@@ -159,6 +159,67 @@ abstract class DurableDataSpec(multiNodeConfig: DurableDataSpecConfig)
     }
   }
 
+  "remove expired" in {
+    runOn(first) {
+
+      val settings = ReplicatorSettings(system)
+        .withGossipInterval(200.millis)
+        .withExpiryKeys(Map(KeyA.id -> 300.millis, KeyC.id -> 300.millis))
```

Comment:
Oh, ofc, here we do expect some form of ms level expiry.

Reply:
it wouldn't be much useful at the ms level, and they are cleared at the gossip tick, so we could probably increase the clock on gossip tick and use that same value throughout all access until next tick (to avoid the currentTimeMillis each time)